server/server.js

```diff
@@ -101,6 +102,14 @@ app.get(`${apiBaseUrl}/employer-form/:id`,
     return res.json(result);
   }));
 
+app.get(`${apiBaseUrl}/coords`,
+  keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health'),
+  asyncMiddleware(async (req, res) => {
+    const { postalCode } = req.query;
```

Comment:
Missing validation

Reply:
I think this actually happens in getPointsFromPostalCodes - which returns null if it doesn't get a hit on the postal code table after handling codes with/without spaces